### PR TITLE
Fix 404 for flower sub-path

### PR DESCRIPTION
**Pull request by andriisoldatenko**
fix https://github.com/astronomer/issues/issues/835

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
image

**Comment by schnie** [comment]
@andriisoldatenko why did we have to split into two `Ingress` objects?

**Comment by schnie** [comment]
@andriisoldatenko why did we have to split into two `Ingress` objects?

**Comment by andriisoldatenko** [comment]
tl;dr it conflicts with airflow one.

there are some issues:
- https://github.com/kubernetes/ingress-nginx/issues/229
- https://github.com/mher/flower/issues/414

File: templates/ingress.yaml [hunk 1]
```diff
@@ -53,17 +49,67 @@ spec:
...
+    - secretName: ~
+            {{- end }}
+      hosts:
+        - {{- include "deployments_subdomain" . | indent 1 }}
+        - {{- include "flower_subdomain" . | indent 1 }}
+  rules:
+    - host: {{- include "deployments_subdomain" . | indent 1 }}
+      http:
+        paths:
+          - path: /
```
Comment (andriisoldatenko) [inline templates/ingress.yaml]: it must be `path: /`

File: templates/ingress.yaml [hunk 2]
```diff
@@ -53,17 +49,67 @@ spec:
...
+{{- end }}
+{{- if and .Values.ingress.enabled (eq .Values.executor "CeleryExecutor") }}
+---
+kind: Ingress
+apiVersion: extensions/v1beta1
+metadata:
+  name: {{ .Release.Name }}-flower-ingress
+  labels:
+    tier: airflow
+    component: airflow-ingress
```
Comment (schnie) [inline templates/ingress.yaml]: @andriisoldatenko maybe `flower-ingress` here?

Comment (schnie) [reply templates/ingress.yaml]: Also, if we change this to `flower-ingress` do we need do do anything on our Grafana or Prometheus stack? I feel like we may have some stuff that looks for this label, but not sure.

Comment (andriisoldatenko) [reply templates/ingress.yaml]: @schnie 
quick look in `github.com/astronomer/astronomer`
```
$ grep --exclude-dir={.git,.idea} -rnw . -e 'airflow-ingress'
```
return nothing.

Comment (andriisoldatenko) [reply templates/ingress.yaml]: fixed component name.

**Comment by andriisoldatenko** [comment]
image
image
fixed few bugs at 480d8e2

**Comment by vishwas-astro** [comment]
@schnie can you review and merge please?

**Pull request merged**